### ISSUE TITLE
Is it possible to remove the R > 3.5.0 requirement?

### ISSUE THREAD

**Issue by gmbecker** [Bioconductor/BiocManager#15]
It is causing problems with switchr, which would like to Suggest it, but cannot have a requirement (even just for checking) on an R version that late.

**Comment by LiNk-NY** [comment]
Hi Gabe, @gmbecker 
`BiocManager` is designed to work with release and devel versions
of Bioconductor thus necessitating R versions >= 3.5.0 moving forward. 
This is not likely to change. 
Regards,
Marcel

**Comment by mtmorgan** [comment]
This seems more worth-while as a way to provide users of earlier versions with a way to install packages without entering URLs, but maybe the idea is for BiocManager::install() indicates that they should instead use BiocInstaller::biocLite(), offering to install BiocInstaller. This would not help switchr, etc.

**Comment by shabbybanks** [comment]
As far as I can tell, R 3.5.0 was released on April 23, 2018, exactly one year ago from today. Many of us are still one or two versions behind because of corporate policy. Bioconductor packages are already difficult enough for me to install. Forcing an R version restriction on `BiocManager` only makes the problem worse.

**Comment by LiNk-NY** [comment]
Hi Shabby, @shabbybanks 
This restriction will be removed in the next release of `BiocManager` but you will
still have to use `BiocInstaller` for older versions of R (< 3.5.0). 

Contact your systems admin to update the R version. 
The R version restriction avoids issues with old versions of Bioconductor. 
`BiocManager` was designed to work with R versions (>= 3.5.0). 

Best,
Marcel

**Comment by shabbybanks** [comment]
Sorry for the curt tone of my previous comment. However, this continues to be a blocker for me. The only solution has been to avoid Bioconductor packages altogether, I am afraid.

**Comment by mtmorgan** [comment]
If you're using R < 3.5, then use
```
source("https://bioconductor.org/biocLite.R")
BiocInstaller::biocLite(your_packages)
```
The fact that BiocManager is only available for R >= 3.5 shouldn't be a blocker for you.

Probably I'm not understanding what your problem is.

**Comment by shabbybanks** [comment]
great, thanks, I will try that.
